docs/personal-dev.md

```diff
@@ -118,17 +118,17 @@ To access the Maestro Azure Postgres DB run
   eval $(make -C dev-infrastructure maestro-miwi-pg-connect)
   psql -d maestro
   ```
-## Observability
+## Tracing
```

Comment:
I'd be in favor of keeping the Observability header (and possibly add more details about the Prometheus/Grafana setup).

Reply:
Sounds good!